pom.xml

```diff
@@ -7,7 +7,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.4.3</version>
+		<version>2.6.6</version>
```

Comment:
Please also check Whitesource vulnearbilities and also update the respective libraries:
wiremock(com.github.jknack:handlebars:4.0.7->Fix: com.github.jknack:handlebars:4.2.0, handlebars - 4.7.7)
gson (2.8.9)
kubernetes-client (5.3.2)

and also migrate from springfox to springdoc (check workflow for inspiration)